### PR TITLE
Improve JS Instrumentation Docs

### PR DESCRIPTION
**Pull request by legalimpurity**
Since [open-telemetry/opentelemetry-js/pull/4298](https://github.com/open-telemetry/opentelemetry-js/pull/4298) SemanticResourceAttributes and SemanticAttributes has individual constants and the constants map constants have been marked as deprecated. Changing documentation to remove deprecated constants.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]
lgtm

**Review by cartermp — APPROVED** [review]
Awesome, thanks

File: content/en/docs/languages/js/instrumentation.md [hunk 1]
```diff
@@ -949,8 +963,8 @@ Finally, you can update your file to include semantic attributes:
 ```javascript
 const doWork = () => {
   tracer.startActiveSpan('app.doWork', (span) => {
-    span.setAttribute(SemanticAttributes.CODE_FUNCTION, 'doWork');
-    span.setAttribute(SemanticAttributes.CODE_FILEPATH, __filename);
+    span.setAttribute(SEMATTRS_CODE_FUNCTION, 'doWork');
+    span.setAttribute(SSEMATTRS_CODE_FILEPATH, __filename);
```
Comment (trentm) [inline content/en/docs/languages/js/instrumentation.md:967]:
```suggestion
    span.setAttribute(SEMATTRS_CODE_FILEPATH, __filename);
```

typo

File: content/en/docs/languages/js/resources.md [hunk 2]
```diff
@@ -92,15 +92,15 @@ configuration option, where you can set them. For example you can update the
 ```javascript
 ...
 const { Resource } = require('@opentelemetry/resources');
-const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+const { SEMRESATTRS_SERVICE_NAME, SEMRESATTRS_SERVICE_NAMESPACE, SEMRESATTRS_SERVICE_VERSION } = require('@opentelemetry/semantic-conventions');
```
Comment (trentm) [inline content/en/docs/languages/js/resources.md:95]: nit

```suggestion
const { SEMRESATTRS_SERVICE_NAME, SEMRESATTRS_SERVICE_NAMESPACE, SEMRESATTRS_SERVICE_VERSION, SEMRESATTRS_SERVICE_INSTANCE_ID } = require('@opentelemetry/semantic-conventions');
```

**Review by trentm — APPROVED** [review]

**Comment by cartermp** [comment]
@legalimpurity could you take care of @trentm's feedback here?

**Pull request merged**